components/game-generator.tsx

```
@@ -2,814 +2,680 @@
+                      <pre className="bg-gray-800 text-gray-200 p-4 rounded-lg overflow-auto h-[400px]">
+                        <code>{currentStage.js}</code>
+                      </pre>
+                    </div>
+                  </TabsContent>
+                </Tabs>
+              </TabsContent>
+
+              <TabsContent value="docs" className="m-0">
+                <div className="bg-white border rounded-lg p-6 prose max-w-none h-[500px] overflow-auto">
+                  {currentStage.md ? (
+                    <div dangerouslySetInnerHTML={{ __html: currentStage.md.replace(/\n/g, "<br>") || "" }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)